### PR TITLE
DP-483

### PR DESCRIPTION
**Pull request by JordanBalesBAO**
- Assigned the trycatch to a variable in order to get it into scope.
- modified the modal popup message to print the flog error message.
- Subset above message to avoid the timestamp printing. 
- Centered the pepfar logo on the login screen

### CODE REVIEW

**Comment by jason-p-pickering** [comment]
I think you need to update the lock and manifest with the new version of datimutils?

**Review by cnemarich — APPROVED** [review]

**Comment by cnemarich** [comment]
Rebased from `dev` to incorporate updated `renv.lock` and `manifest.json` files as well as other changes.

File: server.R [hunk 1]
```diff
@@ -121,7 +120,7 @@ shinyServer(function(input, output, session) {
       sendSweetAlert(
         session,
         title = "Login failed",
-        text = "Please check your username/password!",
+        text = substr(loginAttempt, 28, nchar(loginAttempt)),
```
Comment (jason-p-pickering) [inline server.R:123]: Why 28 and not 29?

Comment (JordanBalesBAO) [reply server.R:123]: I just counted the response I was, and attempted, to make sure it was pliable. Are you see something different?

Comment (jason-p-pickering) [reply server.R:123]: OK, is the response coming from the server itself? This just seems potentially brittle, if we are depending on a certain number of characters.

Comment (JordanBalesBAO) [reply server.R:123]: It is, it feeds from this line "flog.info(paste0("User ", input$user_name, " login failed. ", e$message), name = "datapack")". Do you think maybe a regex would be safer/worthwhile?

Comment (JordanBalesBAO) [reply server.R]: @jason-p-pickering I replaced this with my most recent push. What do you think about this solution?

**Comment by JordanBalesBAO** [comment]
Still shouldn't merge until datimutils DP-483 is merged

File: server.R (hunk 1 above)
Comment (jason-p-pickering) [reply server.R]: Looks better. Does this have any dependencies on a newer version of datimutils? I thought you had added code in a feature branch which gave a bit more information on the nature of the failed login?

**Comment by JordanBalesBAO** [comment]
@jason-p-pickering I updated the renv lock file so should actually show a descriptive error message now.

**Review by cnemarich — APPROVED** [review]

**Pull request merged**